eng/mgmt/automation/generation.yml

```diff
@@ -13,6 +13,8 @@ variables:
   value: '-Dmaven.repo.local=$(MAVEN_CACHE_FOLDER)'
 - name: NodeVersion
   value: '16.x'
```

Comment:
This node version needs to be upgraded as it's deprecated

Reply:
Updated, thanks